tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -306,6 +306,7 @@ def test_it_adds_the_grading_settings(
                     "LISResultSourcedId": f"test_lis_result_sourcedid_{i}",
                     "displayName": f"test_h_display_name_{i}",
                     "userid": f"acct:test_h_username_{i}@TEST_AUTHORITY",
+                    "lmsId": f"test_user_id_{i}",
```

Comment:
I think here `userid` is just a word/concept vs `lmsId` which is more like `LISResultSourcedId`